scheduler/core/components/optimizer/greedymax.py

```diff
@@ -923,9 +924,16 @@ def add(self, night_idx: NightIndex, max_group_info: GroupData | MaxGroup) -> bo
                 # print(f"Adding after_std: {obs.to_unique_group_id} {obs.id.id}")
                 n_slots_filled = self._add_visit(night_idx, obs, max_group_info, best_interval, n_slots_filled)
 
+            # Inactivate any standards not used
+            for obs in part_obs:
+                if obs not in standards:
+                    print(f'Deactivating observation {obs.id.id}')
+                    obs.status = ObservationStatus.INACTIVE
+
             # TODO: Shift to remove any gaps in the plan?
 
             # Re-score program (pseudo time accounting)
+            print(f'Update score')
```

Comment:
We should probably eliminate the debugging print here, but I'll put that in my next PR.